packages/core-mobile/ios/AvaxWallet/AvaxWallet.entitlements

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>aps-environment</key>
+	<string>development</string>
```

Comment:
how are we gonna set it to production?

Reply:
this will be get updated based on the provisioning profile when app is built in xcode